src/com/janek/basics/CalcPremium.java

```diff
@@ -0,0 +1,85 @@
+package com.janek.basics;
+
+import static java.lang.Double.parseDouble;
+import static java.lang.String.format;
+import static java.lang.System.*;
+import static com.janek.basics.math.MathUtil.*;
+
+public class CalcPremium {
+    public static final String PLUS_SYMBOL = "+";
+    public static final String PLUS_STRING = "dodać";
+    public static final String MINUS_SYMBOL = "-";
+    public static final String MINUS_STRING = "odjąć";
+    public static final String X_SYMBOL = "x";
+    public static final String X_STRING = "razy";
+    public static final String SLASH_SYMBOL = "/";
+    public static final String SLASH_STRING = "podzielić-przez";
+
+    public static void main(String[] args) {
+
+        validate(args);
+
+        double one = parseDoubleSafely(args[0]);
+        String symbol = args[1];
+        double two = parseDoubleSafely(args[2]);
+
+        checkAddition(symbol, one, two);
+        checkSubtraction(symbol, one, two);
+        checkMultiplication(symbol, one, two);
+        checkDivision(symbol, one, two);
+        checkPropriety(symbol);
```

Comment:
Let's improve this a bit!
IMHO the better idea is adding a new method to your `MathUtil` class, e.g:
`public static double doCalculation(double number1, String operation, double number2)`
Please implement this method using *the switch statement* (https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html)